common/packages.tex

```diff
@@ -87,7 +87,7 @@
 \usepackage{pdflscape}
 
 \usepackage{fancyvrb}
-\usepackage[outputdir=build]{minted}
+\usepackage{minted}
 \setminted{
```

Comment:
New version of minted now respects the output-dir? Or why is this needed?

Reply:
Yes.
`! Package minted Error: Package option "outputdir" is no longer needed with minted v3+; the output directory is automatically detected for TeX Live 2024+, and the environment variable TEXMF_OUTPUT_DIRECTORY can be set manually in other cases.

See the minted package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.278 \ProcessPgfOptions{/minted/pkg}`